dev-libs/rocm-comgr/rocm-comgr-5.0.2.ebuild

```diff
@@ -0,0 +1,53 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake prefix
+
+if [[ ${PV} == *9999 ]] ; then
+	EGIT_REPO_URI="https://github.com/RadeonOpenCompute/ROCm-CompilerSupport/"
+	inherit git-r3
+	S="${WORKDIR}/${P}/lib/comgr"
+else
+	SRC_URI="https://github.com/RadeonOpenCompute/ROCm-CompilerSupport/archive/rocm-${PV}.tar.gz -> ${P}.tar.gz"
+	S="${WORKDIR}/ROCm-CompilerSupport-rocm-${PV}/lib/comgr"
+	KEYWORDS="~amd64"
+fi
+
+PATCHES=(
+	"${FILESDIR}/${PN}-4.5.2-dependencies.patch"
+)
+
+DESCRIPTION="Radeon Open Compute Code Object Manager"
+HOMEPAGE="https://github.com/RadeonOpenCompute/ROCm-CompilerSupport"
+LICENSE="MIT"
+SLOT="0/$(ver_cut 1-2)"
+
+RDEPEND=">=dev-libs/rocm-device-libs-${PV}
+	>=sys-devel/llvm-roc-${PV}:="
+DEPEND="${RDEPEND}"
+
+CMAKE_BUILD_TYPE=Release
+
+# PATCHES=( "${FILESDIR}"/${PN}-5.0-rocm_path.patch )
+
+src_prepare() {
+	sed '/sys::path::append(HIPPath/s,"hip","",' -i src/comgr-env.cpp || die
+	sed '/sys::path::append(LLVMPath/s,"llvm","lib/llvm/roc",' -i src/comgr-env.cpp || die
+	sed '/Args.push_back(HIPIncludePath/,+1d' -i src/comgr-compiler.cpp || die
+	sed '/Args.push_back(ROCMIncludePath/,+1d' -i src/comgr-compiler.cpp || die # ROCM and HIPIncludePath is now /usr, which disturb the include order
+	eapply "${FILESDIR}"/${PN}-5.0-rocm_path.patch
+	eprefixify src/comgr-env.cpp
```

Comment:
try `eapply $(prefixify_ro "${FILESDIR}"/${PN}-5.0-rocm_path.patch)`

Reply:
> Is `/usr/lib64/libhsakmt.so.1.0.6` not used by `hip` anymore? I see hip still links to it. Why remove it from DEPEND?

Emm, hip's CMakeLists.txt doesn't have any related information about hsakmt; only roctracer requires libhsakmt.so. With `USE=-profile`, there is no sign of `hsakmt` in `build.ninja`. If force removed roct-thunk-interface, the compile will not fail (but hip program fails at runtime). I guess the compiler is automatically link libhsakmt.so even if libhsakmt.so is not present.